plugins/modules/nsxt_uplink_profiles.py

```diff
@@ -226,6 +226,10 @@ def check_for_update(module, manager_url, mgr_username, mgr_password, validate_c
     if existing_profile.__contains__('transport_vlan') and profile_params.__contains__('transport_vlan') and \
         existing_profile['transport_vlan'] != profile_params['transport_vlan']:
         return True
+    if existing_profile.__contains__('lags') and not profile_params.__contains__('lags'):
```

Comment:
Bugs Fixed - 2632169 - [Ansible]deletion of all 'lags' and creation of  'lags' is not supported by 'nsxt_uplink_profiles' ansible module